### ISSUE TITLE
Bug: sort issues with properties.

### ISSUE THREAD

**Issue by lifeiscontent** [brigade/scss-lint#82]
there is an issue with scss-lint, properties like this should not be sorted.

also, I don't agree with all of this, but most of it is good.

http://mdo.github.io/code-guide/

there should be a config option for both single quotes and double quotes linting.

some people prefer double quotes (I do).

but I also love the benefits that scss-lint gives me.

``` scss
border-radius: 4rem; // this should not be put at the bottom.
border-bottom-right-radius: 0;
border-bottom-left-radius: 0;
```

the work around I've done to get scss-lint to stop complaining about the border-radius issue is by expanding out the other 2 properties (border-top-right-radius, etc). and setting those. So  I guess you could tell people to do that instead.

Thanks!
